libraries/AP_NavEKF2/AP_NavEKF2_MagFusion.cpp

```diff
@@ -1174,3 +1213,111 @@ void NavEKF2_core::recordMagReset()
     yawInnovAtLastMagReset = innovYaw;
 }
 
+// Reset states using yaw from EKF-GSF and velocity and position from GPS
+bool NavEKF2_core::EKFGSF_resetMainFilterYaw()
+{
+    // Don't do a reset unless permitted by the EK3_GSF_USE and EKF3_GSF_RUN parameter masks
```

Comment:
should be EKF2 in comment

Reply:
I've fixed this in the follow-up commits at the end of the branch linked in the comments.